foundation/plugins/org.jboss.tools.foundation.ui/src/org/jboss/tools/foundation/ui/util/BrowserUtility.java

```diff
@@ -37,7 +75,7 @@
 	 * @param log
 	 *            the log provider to log against if an error occurred.
 	 */
-	public void checkedCreateInternalBrowser(String url, String browserId, String pluginId, ILog log) {
+	public static void checkedCreateInternalBrowser(String url, String browserId, String pluginId, ILog log) {
```

Comment:
Verified:  before this patch, BrowserUtility had only one method in it, and was not static. It seems Denis' patch added a half-dozen or more methods, all of which were static, and this method was changed to static to be more uniform and for re-use by the newer signatures that were static and may have needed to use the existing method.